### PR TITLE
chore: limit concurrency on the `transform.sh` script by default

### PR DESCRIPTION
**Pull request by njlynch**
The transform script, as its last step, runs `lerna run --stream build+test` (by
default). lerna run tries to organize the builds topologically, but with the
alpha packages, they're (almost) all completely independently. This leads to it
attempting to build and test all ~25 alpha packages at once, killing (or
severely crippling) most machines.

Setting a default concurrency limit to reduce resource contention and make this
script executable in environments that aren't our ultra-powerful build machines.

(Open to arguments the default should be 6 or 8 or 2 rather than 4.)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by otaviomacedo — APPROVED** [review]

File: scripts/transform.sh [hunk 1]
```diff
@@ -59,5 +59,6 @@ cd "$individual_packages_folder"
 createSymlinks "$individual_packages_folder"
 
 if [ "$skip_build" != "true" ]; then
-  PHASE=transform yarn lerna run --stream $runtarget
+  CONCURRENCY=${CONCURRENCY:-4}
```
Comment (nija-at) [inline scripts/transform.sh:62]: Should we call this `TRANSFORM_CONCURRENCY` or some other just to make sure it doesn't conflict with some other library's env var?

Comment (nija-at) [inline scripts/transform.sh:62]: Should we raise the value for the cdk build or is it 4 even now?

**Review by nija-at — APPROVED** [review]

File: scripts/transform.sh (hunk 1 above)
Comment (njlynch) [reply scripts/transform.sh:62]: Sure, can do.

Comment (njlynch) [reply scripts/transform.sh:62]: The value is currently ♾  for the cdk build. What would you propose we set it to?

Comment (nija-at) [reply scripts/transform.sh]: What is the actual translate to today? The number of alpha modules? 
Perhaps that's a good start, given that the build works there and we don't want to slow it down any further.

I'm not fussed here, if you want to leave it at 4.

Comment (njlynch) [reply scripts/transform.sh]: The number is 26 today, which I think it unreasonably high.

I'm going to look at the PR build timings (compared to other PR builds) and see how much 4 as the default slows it down. I also just pushed a change to make the default 8, and I'll see how that goes. I suspect going much higher than 8 isn't going to be much faster, due to resource contention, but it's worth a look.

Comment (njlynch) [reply scripts/transform.sh]: With concurrency of 8, we're within a minute of the original (unbounded) run time. I think that's a fair compromise.

**Pull request merged**